accord-core/src/main/java/accord/local/RedundantStatus.java

```diff
@@ -159,6 +166,7 @@ public static class SomeStatus
         public static final SomeStatus MAJORITY_APPLIED_ONLY = oneSlow(MAJORITY_APPLIED);
         public static final SomeStatus SHARD_APPLIED_ONLY = oneSlow(SHARD_APPLIED);
         public static final SomeStatus GC_BEFORE_AND_LOCALLY_APPLIED = multi(GC_BEFORE, LOCALLY_APPLIED);
+        public static final SomeStatus SHARD_UNSAFE_BEFORE = oneSlow(UNSAFE_BEFORE);
```

Comment:
definitely not SHARD unsafe, in this class SHARD refers to a whole shard property. We mean LOCALLY_UNSAFE or perhaps better would be LOCALLY_CORRUPT or LOCALLY_LOST or LOCALLY_INCOMPLETE. This should be consistent with the Property name, and be suffixed with _ONLY here.

Reply:
Renamed to LOCALLY_INCOMPLETE